apps/meteor/tests/e2e/page-objects/fragments/home-flextab-members.ts

```diff
@@ -26,6 +26,18 @@ export class HomeFlextabMembers {
 		await this.page.getByRole('dialog').getByRole('button').first().click();
 	}
 
+	async ignoreUser(username: string) {
+		await this.page.locator(`[data-qa="MemberItem-${username}"]`).click();
+		await this.page.locator('role=button[name="More"]').click();
+		await this.page.locator('role=menuitem[name="Ignore"]').click();
+	}
+
+	async unignoreUser(username: string) {
+		await this.page.locator(`[data-qa="MemberItem-${username}"]`).click();
+		await this.page.locator('role=button[name="More"]').click();
+		await this.page.locator('role=menuitem[name="Unignore"]').click();
+	}
+
```

Comment:
Since the function body is almost identical we could extract that "shared" to a new private function, something like:

```
private async ignoreUserAction(action: string, username: string): {
        await this.page.locator(`[data-qa="MemberItem-${username}"]`).click();
	await this.page.locator('role=button[name="More"]').click();
	await this.page.locator(`role=menuitem[name="${action}"]`).click();
}

async ignoreUser(username: string) {
	await this.ignoreUserAction('Ignore', username);
}

async unignoreUser(username: string) {
	await this.ignoreUserAction('Unignore', username);
}
```

Reply:
Awesome, thank you